provider/win_driver_utils.py

```diff
@@ -194,7 +194,119 @@ def copy_file_to_samepath(session, test, params):
                        " the detailed info:\n%s." % output)
 
 
-def memory_leak_check(vm, test, params):
+def enable_driver(vm, test, cmd, device_id):
+    """
+    Enable driver.
+
+    :param vm: vm object
+    :param test: kvm test object
+    :param cmd: Driver enable
+    :param device_id: Device id in windows guest
+    """
+    session = vm.wait_for_login()
+    cmd = cmd.replace("DEVICE_ID", device_id)
+    cmd = utils_misc.set_winutils_letter(session, cmd)
+    status, output = session.cmd_status_output(cmd)
+    if status != 0:
+        test.fail("failed to enable driver, %s" % output)
+
+
+def disable_driver(vm, test, cmd, device_id):
+    """
+    Disable driver.
+
+    :param vm: vm object
+    :param test: kvm test object
+    :param cmd: Driver disable command
+    :param device_id: Device id in windows guest
+    """
+    session = vm.wait_for_login()
+    cmd = cmd.replace("DEVICE_ID", device_id)
+    cmd = utils_misc.set_winutils_letter(session, cmd)
+    status, output = session.cmd_status_output(cmd)
+    if status != 0:
+        if "reboot" in output:
+            vm.reboot()
+        else:
+            test.fail("failed to disable driver, %s" % output)
+
+
+def get_device_id(vm, test, params):
+    """
+    Get device id from guest.
+
+    :param vm: vm object
+    :param test: kvm test object
+    :param params: the dict used for parameters
+    """
+    device_name = params["device_name"]
```

Comment:
@vivianQizhu , sorry, I just found  we can use "device_name" and  "device_hwid" in this module directly. So It can be optimized  here，let me  update it.